### PR TITLE
Feature/create scaffolding

### PR DESCRIPTION
**Pull request by Malet**

### CODE REVIEW

File: lib/conduit/workspace/workspace.ex [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+      iex> change_pipeline_group(pipeline_group)
+      %Ecto.Changeset{source: %PipelineGroup{}}
+
+  """
+  def change_pipeline_group(%PipelineGroup{} = pipeline_group) do
+    PipelineGroup.changeset(pipeline_group, %{})
+  end
+
+  alias Conduit.Workspace.Pipeline
```
Comment (IanLuites) [inline lib/conduit/workspace/workspace.ex:105]: I think the auto generate does this if you generate more within the same context.

Maybe add a comment like `### Pipeline ###` and move the `alias` to the top.

File: lib/conduit_web/templates/pipeline/show.html.eex [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<h2>Show Pipeline</h2>
+
+<ul>
+
+  <li>
+    <strong>Name:</strong>
+    <%= @pipeline.name %>
+  </li>
+
```
Comment (IanLuites) [inline lib/conduit_web/templates/pipeline/show.html.eex:9]: Empty line? (also line 4)

**Review by IanLuites — APPROVED** [review]
For now this works perfectly, we might want to look into using `defdelegate` if it grows.

File: lib/conduit_web/templates/pipeline/show.html.eex (hunk 2 above)
Comment (Malet) [reply lib/conduit_web/templates/pipeline/show.html.eex:9]: Will clean all these up with credo after I've got the Jenkins pipeline set up

**Pull request merged**